src/Data/LLVM/BitCode/Parse.hs

```diff
@@ -57,8 +56,10 @@ newtype Parse a = Parse
   } deriving (Functor, Applicative, MonadFix)
 
 instance Monad Parse where
+#if !MIN_VERSION_base(4,11,0)
```

Comment:
If we want to avoid this CPP, we could alternatively raise the lower version bounds on `base` to `4.11`—that is, require GHC 8.4 or later to build this library. I can't think of any projects where we use `llvm-pretty-bc-parser` to support versions of GHC older than this, but double-check me on that.

Reply:
Usually we endeavour to support 3 GHC version; with a major change like 9 (and especially the concerns around 9.0) we extended that a bit, but even under those relaxed conditions, I don't think we're too concerned with 8.4 any longer.

That said, I'm not sure that removing this right now buys us much because we still have the 4.13 base (GHC 8.8) CPP for MonadFail and I don't think we're quite ready to drop 8.8 support yet.

Reply:
> That said, I'm not sure that removing this right now buys us much because we still have the 4.13 base (GHC 8.8) CPP for MonadFail and I don't think we're quite ready to drop 8.8 support yet.

Alright. If you're content, then so am I.